packages/contentstack-config/src/services/mfa/mfa-service.interface.ts

```diff
@@ -0,0 +1,12 @@
+import { MFAConfig } from './mfa.types';
+
+export interface IMFAService {
```

Comment:
I would suggest move all services code to simple utils folder, or move to contentstack uitlities use it accross, it seems this logic is everywhere, maintenance would be difficult